shepherd/sheep/__init__.py

```diff
@@ -1,6 +1,6 @@
 from .base_sheep import BaseSheep
 from .docker_sheep import DockerSheep
 from .bare_sheep import BareSheep
-from .errors import SheepError, SheepConfigurationError
+from ..errors.sheep import SheepError, SheepConfigurationError
 
 __all__ = ['BaseSheep',  'DockerSheep', 'BareSheep', 'SheepError', 'SheepConfigurationError']
```

Comment:
In `comm` you removed the errors from `__all__` while here you keep them. I would opt for the first option, anyways it should be consistent.